src/microbe_stage/PatchMapNameGenerator.cs

```diff
@@ -40,7 +40,7 @@ public string PatchName
     public string InternalName { get; set; } = null!;
 
     /// <summary>
-    ///   Generates and returns a new patch name
+    ///   Generates and returns a new region name
```

Comment:
I think the variables within this method need renaming now as well. There's at least PatchName on line 84